js/feature/wigTrack.js

```diff
@@ -137,6 +138,66 @@ WigTrack.prototype.draw = function (options) {
         baselineColor = IGVColor.addAlpha(self.color, 0.1);
     }
 
+    const drawGuideLines = function (options) {
+        if (self.config.hasOwnProperty('guideLines')) {
+            for (let line of self.config.guideLines) {
+                if (line.hasOwnProperty('color') && line.hasOwnProperty('y') && line.hasOwnProperty('dotted')) {
+                    options.context.setLineDash([5,5]);
```

Comment:
I need to change this so that it only set line dash if the line.dotted property is true;

Reply:
Fixed in later commit